resources/definitions/fdmprinter.def.json

```diff
@@ -3404,27 +3404,44 @@
                 },
                 "support_skip_some_zags":
                 {
-                    "label": "Skip Some ZigZags Connections",
-                    "description": "Skip some ZigZags connections to make the support structure easier to break.",
+                    "label": "Skip Some Support Line Connections",
+                    "description": "Skip some support line connections to make the support structure easier to break away. This setting is applicable to the Zig Zag support infill pattern.",
                     "type": "bool",
                     "default_value": false,
                     "enabled": "support_enable and (support_pattern == 'zigzag')",
                     "limit_to_extruder": "support_infill_extruder_nr",
                     "settable_per_mesh": false,
                     "settable_per_extruder": true
                 },
-                "support_zag_skip_count":
+                "support_skip_zag_per_mm":
                 {
-                    "label": "ZigZag Connection Skip Count",
-                    "description": "Skip one in every N connection lines to make the support structure easier to break.",
-                    "type": "int",
-                    "default_value": 6,
-                    "minimum_value": "1",
-                    "minimum_value_warning": "3",
+                    "label": "Support Line Skip Frequency",
+                    "description": "To skip one connection line in every N mm to make the support structure easier to break away.",
+                    "type": "float",
+                    "value": "support_line_distance * 5",
+                    "minimum_value": "0",
+                    "minimum_value_warning": "support_line_distance",
                     "enabled": "support_enable and (support_pattern == 'zigzag') and support_skip_some_zags",
                     "limit_to_extruder": "support_infill_extruder_nr",
                     "settable_per_mesh": false,
-                    "settable_per_extruder": true
+                    "settable_per_extruder": true,
+                    "children":
+                    {
+                        "support_zag_skip_count":
+                        {
+                            "label": "Support Chunk Size",
```

Comment:
I think this new terminology ('support chunks') is good, but it is now applied inconsequently.
I propose:
Skip Some Support Line Connections -> Break Up Support In Chunks   (not sure about this one, please discuss with Ruben)
Support Line Skip Frequency -> Support Chunk Size
Support Chunk Size -> Support Chunk Line Count